src/containers/BooksForm.js

```diff
@@ -0,0 +1,18 @@
+const BooksForm = () => {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
## Linter Disabled ⚠️ 
- [ ] There is an easy fix for this rule instead of the disable option. Let's lift this and improve on our unused variables usage, thank you!

Reply:
I think I will fix this in the next milestone, when I will start to use that array @christianotieno, or if you have a suggestion , pls let me know